src/pages/ProfilesExplorerView/domain/variables/FiltersVariable/FiltersVariable.tsx

```diff
@@ -47,18 +46,11 @@ export class FiltersVariable extends AdHocFiltersVariable {
 
   static Component = ({ model }: SceneComponentProps<AdHocFiltersVariable & { onChangeQuery?: any }>) => {
     const styles = useStyles2(getStyles);
+
     const { key } = model.useState();
-    const [, setQuery] = useQueryFromUrl();
 
     const query = useBuildPyroscopeQuery(model, key as string);
 
-    useEffect(() => {
```

Comment:
By removing this code and the one below in `useToggleSidePanel` we are getting rid of the `query` URL search parameters.